benchmark/primitives/primitives_test.go

```diff
@@ -256,7 +256,28 @@ func BenchmarkAtomic32BValueStore(b *testing.B) {
 	b.StopTimer()
 }
 
+func BenchmarkAtomicPointerStore(b *testing.B) {
```

Comment:
Oh, I meant [`atomic.StorePointer`](https://golang.org/pkg/sync/atomic/#StorePointer), not `atomic.Store` using a pointer type.  :smile: